controllers/storagecluster/external_resources.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"crypto/sha512"
 	"encoding/json"
+	inbuilterrors "errors"
```

Comment:
```suggestion
	"errors"
```
Keep this as `errors` and rename the other to `apierrors`.

Reply:
I removed this and using fmt itself for error.